src/mudlet.h

```diff
@@ -326,8 +326,8 @@ class mudlet : public QMainWindow, public Ui::main_window
         // Takes two extra arguments (profile name, mapFileName) that returns
         // the pathFile name for any map file:
         profileMapPathFileName,
-        // Takes one extra argument (profile name) that returns the pathFile
-        // name for the downloaded IRE Server provided XML map:
+        // Takes one extra argument (profile name) that returns the file
```

Comment:
I use the word `pathFile` name with deliberation here - it means the path AND the file in one `QString` name - your use of `file location` is more ambiguous is it 1) a directory, 2) a directory and a file or 3) just a file?

Reply:
It's file location = location of the file, I don't think anyone will get confused.

Reply:
Actually Dentists have but it means something else entirely (a rotary file that is used to create the canal for a *root-canal* filling - *Ouch*!) - anyhow it is shorter than `pathAndFileName` and you have previously expressed dissatisfaction at other of my `longAndDescriptiveButPerhapsTooVerboseVariableName`s - whereas the term I use here and elsewhere is handily specific as I see it. :wink: